docs/source/fr/autoclass_tutorial.md

```diff
@@ -0,0 +1,142 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Chargement d'instances pré-entraînées avec une AutoClass
+
+Avec autant d'architectures Transformer différentes, il peut être difficile d'en créer une pour votre point de contrôle. Dans l'idée de créer une librairie facile, simple et flexible à utiliser, 🤗 Transformers fournit une `AutoClass` qui infère et charge automatiquement l'architecture correcte à partir d'un point de contrôle donné. La fonction `from_pretrained()` vous permet de charger rapidement un modèle pré-entraîné pour n'importe quelle architecture afin que vous n'ayez pas à consacrer du temps et des ressources à l'entraînement d'un modèle à partir de zéro. Produire un tel code indépendant du point de contrôle signifie que si votre code fonctionne pour un point de contrôle, il fonctionnera avec un autre point de contrôle - tant qu'il a été entraîné pour une tâche similaire - même si l'architecture est différente.
```

Comment:
point de control is necessarly wrong 😅 let's check what other translation use instead of this but it sounds really off.

Reply:
Agreed, I modified it to `poids` (per your comment on my previous question). I also added in parentheses the standard terms in English as it may be possible that they are used as-is in French.